kuma/core/models.py

```diff
@@ -34,6 +37,21 @@ def update(self, **kw):
                                           created=False)
 
 
+class IPBan(models.Model):
+    ip = models.GenericIPAddressField()
+    created = models.DateTimeField(default=datetime.now, db_index=True)
```

Comment:
Let's use `django.utils.timezone.now` for the default in preparation to make kuma timezone aware (a feature in current Django)